examples/repl/cmdline.rs

```diff
@@ -475,6 +477,27 @@ pub async fn cmdline(context: Context, line: &str, chat_id: &mut ChatId) -> Resu
             .await?;
             println!("Exported to {}.", dir.to_string_lossy());
         }
+        "send-backup" => {
+            let dir = dirs::home_dir().unwrap_or_default();
```

Comment:
would a named temporary directory be better?  you could print the name to help debugging.  This is kind of unconditionally overwriting files in the home directory now IIUC.

Reply:
fixed